src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -114,14 +112,7 @@ export class AttachmentUnitFormComponent implements OnInit, OnChanges {
     }
 
     get isSubmitPossible() {
-        return !(this.form.invalid || this.fileUploadErrorMessage || this.fileName === this.fileNamePlaceholder);
-    }
-
-    // will be called from parent component to set the form error when the file upload failed
-    setFileUploadError(errorMessage: string) {
```

Comment:
❌ Required Change: File Upload Error Messages because of unsupported type or file size are not displayed any more. See screenshot how it looks on develop:

![image](https://user-images.githubusercontent.com/29718932/186999232-3bbba1f0-bc31-47aa-873d-48ad11a0c336.png)

On this branch: Internal Server Error and endless loading loop
![image](https://user-images.githubusercontent.com/29718932/186999370-9e5e2fb4-5c49-40e1-8800-f8c59bbacad3.png)

Reply:
It should work now.
I decided to now use the new error system for the errors coming from the server instead, but I misunderstood what `complete` does. I interpreted it as `finalize`, which it isn't.

Reply:
Great! Could you please test it if it works now :) ?

Reply:
Yeah, I did already :D I always write "should work" for some reason. Sorry :D